lib/PerconaTest.pm

```diff
@@ -78,6 +83,19 @@ eval {
    $sandbox_version = $v if $v;
 };
 
+our $source_name = 'source';
+our $source_status = 'binary log';
+our $source_reset = 'binary logs and gtids';
+our $source_change = 'replication source';
+our $replica_name = 'replica';
+if ( $sandbox_version lt '8.1' || ( $ENV{FORK} || "" eq 'mariadb' ) ) {
```

Comment:
works in an unintended way (PT-2390). I listed this code path there for review. I'm still commenting since it's a different condition
10.x will return true on `$sandbox_version lt '8.1'`, so the second part on the fork is not used.
```
$ echo $sandbox_version $FORK
10.4 mysql
$ perl -e "if ( $sandbox_version lt '8.1' || ( $ENV{FORK} || '' eq 'mariadb' ) ) { print('works') }"
works
```
As long as mysql 10 does not exist it still does what we want

Reply:
Will ignore it for now, fix later.